pytorch_lightning/loggers/mlflow.py

```diff
@@ -11,16 +10,20 @@
     import mlflow
     from mlflow.tracking import MlflowClient
     _MLFLOW_AVAILABLE = True
-except ImportError:  # pragma: no-cover
+except ModuleNotFoundError:  # pragma: no-cover
```

Comment:
maybe have both?

Reply:
ModuleNotFoundError is a subclass of ImportError, so we only catch it when the module is not installed. I this is a bit better practice :)